### PR TITLE
Second Assignment

### PR DESCRIPTION
**Pull request by apoorvamoodbidri**

### CODE REVIEW

File: iOS4-Assessment-2.playground/Contents.swift [hunk 1]
```diff
@@ -6,61 +6,131 @@
...
 // 2. Write a variable to hold the duration of a movie
 //    in minutes and set its value
 
+var durationOfMovie = 125
+
 // 3. Write a variable to hold whether it is daytime
 //    and set it to reflect that it is nighttime
 
+var isNightTime = true
+
```
Comment (jeffreybergier) [inline iOS4-Assessment-2.playground/Contents.swift]: the variable name should reflect whether it "is daytime"

``` swift
var isDaytime = false
```

File: iOS4-Assessment-2.playground/Contents.swift [hunk 2]
```diff
@@ -6,61 +6,131 @@
...
 // 5. Write a constant to store your favorite holiday
 
-// 6. Print out a message to the console saying 
+let myFavoriteHoliday = "Thanksgiving"
+
+// 6. Print out a message to the console saying
 //    "My favorite holiday is ___" 
 //    (fill in the blank using the variable from #5)
 
+print ( "My favorite holiday is " , myFavoriteHoliday )
```
Comment (jeffreybergier) [inline iOS4-Assessment-2.playground/Contents.swift]: I didn't even know this worked in swift. However, normally you would accomplish this question with String Interpolation:

``` swift
print("My favorite holiday is \(myFavoriteHoliday)")
```

**Comment by jeffreybergier** [comment]
Exceed

Everything is correct. There were just two minor issues:
Naming issue in # 3
Syntax issue in # 6

**Comment by apoorvamoodbidri** [comment]
Thank you !

**Pull request closed without merging**